ftw/linkchecker/linkchecker.py

```diff
@@ -46,14 +50,19 @@ def get_uri_response(external_link_obj, timeout):
     return external_link_obj
 
 
-def limit_cpu():
-    os.nice(19)
-
-
 def work_through_urls(external_link_objs, timeout_config):
+    # get correct logger
+    logger = logging.getLogger('ftw.linkchecker')
+    logger_name = logger.name
+
+    # prepare worker function and pool
     part_get_uri_response = partial(get_uri_response, timeout=timeout_config)
-    pool = Pool(initializer=limit_cpu)
+    pool = PoolWithLogging(processes=cpu_count(), logger_name=logger_name)
+
+    # run tasks using pool.map
```

Comment:
You can remove that comment, doesn't help to understand the code better.

Reply:
?

Reply:
Yes thats absolutely true. It sometimes helps me create better visible sections so I forget that the comment itself does not add any value...